starter-code/css/main.css

```diff
@@ -6,3 +6,122 @@ Green: #00B172
 White: #FFF
 
 */
+body {
+  height: 75%:
+}
+h1 {
+  font-size: 5em;
+}
+
+h2 {
+  font-size: 2.5em;
+  text-align: center;
+  border: 2px #3CB371;
+  border-bottom-style: solid;
+  width: 30%;
+  margin: 50px auto;
+}
+
+h3 {
+  color: #3CB371;
+  font-size: 1.3em;
+  font-weight: 900;
+}
+h4 {
+  font-size: 2.5em;
+  color: #FFF;
+  border: 2px #FFF;
+  border-bottom-style: solid;
+}
+
+h5 {
+  color: #FFF;
+  font-size: 1.3em;
+}
+
+p {
+  color: #FFF;
+  font-size: 1.2em;
+}
+
+quote {
+  font-size: 2em;
+}
+
+  @keyframes bounce {
+    100% {
+      margin-top: 100px;
+    }
+  }
+
+  .animation-logo:hover {
+    animation: bounce 1s;
+    animation-iteration-count: infinite;
+  }
+  .nav-style{
+    background-color: #FFF;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    padding: 20px;
+  }
+
+  .nav-style > ul {
+    float: right;
+    list-style: none;
+    display: inline-flex;
+  }
+
+
+
+  .nav-style > img {
+    width: 10%;
+    float: left;
+  }
+  .div-central {
+    background: url("/home/valentin/code/lab/lab-css-spotify-clone/starter-code/images/landing.jpg");
```

Comment:
Use a relative path instead. When you use absolute paths it only works on your computer; as soon as you put your page online everything will break.